composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0",
+        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0|~6.1.0|~6.2.0",
```

Comment:
Can we just switch this to `"~5.5|~6.0"`?  Leaving off the 3rd number should allow it to match "5.*" and "6.*"